environment.yml

```diff
@@ -3,11 +3,12 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - scipy>=0.11
-  - pandas
+  - geopandas
   - libpysal
-  - scikit-learn
   - mapclassify
-  - geopandas
-  - numba
   - matplotlib
+  - numba
+  - pandas
+  - scikit-learn
+  - scikit-learn<1.1
```

Comment:
Do we need this then?
```suggestion
```

Reply:
Nope, that snuck back in during pull/merge.